pkgs/development/libraries/openldap/default.nix

```diff
@@ -16,9 +16,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ openssl cyrus_sasl db groff libtool ];
 
   configureFlags =
-    [ "--enable-overlays"
+    [
       "--disable-dependency-tracking"   # speeds up one-time build
+      "--disable-ndb"
+      "--disable-perl"
+      "--disable-sql"
+      "--enable-backends=mod"
```

Comment:
> have to specify the path of modules-dir

No you haven't. OpenLDAP modules are immediately available, because it knows its libdir (`$out/lib/...`)